meeting-notes/2024/2024-01-29_skimage2-meeting.md

```diff
@@ -0,0 +1,109 @@
+# Meeting notes "Coordinating skimage2" 
+
+- June 29, 2024 at 20:00 UTC
+- [Video meeting room](https://meet.evolix.org/skimage-meeting)
+- [Discussion thread](https://discuss.scientific-python.org/t/yet-another-pathway-towards-skimage-v2/859/6)
+
+**Participants:** Lars, Egor, Marianne, Josh, Jarrod, Stéfan, Juan
+
+## Agenda outline
+
+- Short presentation of latest transition plan (by Lars)
+- **Focus: Discuss coordination**
+- Optional: Discuss transition plan in detail
+- Optional: Discuss typing scikit-image
+
+## Coordination
+
+- *Realistically, how much resources do we have?*
+  - Paid hours left according to EOSS5 budget (Jan 2024):
+    - Lars: ~1200 h
+    - Marianne: ~250 h
+
+- *How will we communicate about the work?*
+  - Weekly dev update from Lars on discuss.scientific-python.org community list
+
+- *How should work be prioritized?*
+  - There's a new tag: [`🥾 Path to skimage2`](https://github.com/scikit-image/scikit-image/labels/%3Ahiking_boot%3A%20Path%20to%20skimage2)
+  - Use GitHub project (new) or rather a milestone? 
+  - Requesting reviews?
+  - ...?
+
+- *How do we speed-up consensus-seeking and decision making for the new API?*
+  - [Clarify objection period for lazy consensus in SKIP 1 #7020](https://github.com/scikit-image/scikit-image/pull/7020)
+  - ...?
+
+- *How can paid maintainers help with bottlenecks?*
+  - Coordinate consensus seeking periods and call votes?
```

Comment:
```suggestion
  - Coordinate consensus-seeking periods and call votes?
```
I don't remember whether we retained this option; if so, we can remove the question mark.